examples/4-apollo-tutorial/client/src/models/RootStore.base.js

```diff
@@ -15,14 +15,18 @@ import { UserModel } from "./UserModel"
 import { userModelPrimitives, UserModelSelector } from "./UserModel.base"
 
 
+
+
+
+
 /**
 * Store, managing, among others, all the objects received through graphQL
 */
 export const RootStoreBase = MSTGQLStore
   .named("RootStore")
-  .extend(configureStoreMixin([['LaunchConnection', () => LaunchConnectionModel], ['Launch', () => LaunchModel], ['Mission', () => MissionModel], ['Rocket', () => RocketModel], ['User', () => UserModel]], ['Launch', 'Rocket', 'User']))
+  .extend(configureStoreMixin([['LaunchConnection', () => LaunchConnectionModel], ['Launch', () => LaunchModel], ['Mission', () => MissionModel], ['Rocket', () => RocketModel], ['User', () => UserModel]], ['Launch', 'Rocket', 'User'], "js"))
   .props({
-    launchs: types.optional(types.map(types.late(() => LaunchModel)), {}),
+    launches: types.optional(types.map(types.late(() => LaunchModel)), {}),
```

Comment:
I don't this example has been ran in awhile. The `launches` and `"js"` changes above did not come from the new code generation.